notebooks/MAST/ultrafaint_dwarf_galaxy_search/ultrafaint_dwarf_galaxy_search.ipynb

```diff
@@ -0,0 +1,1247 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    obj = "mean" #we will look at the mean data objects, dr1 only has mean and stack, dr2 has more options
</pre><p>Could you explain what "mean" and "stack" mean? Perhaps including some explanation about the catalog structure would be beneficial.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>